packages/sdk/src/types.ts

```diff
@@ -1425,6 +1421,12 @@ export interface AccountTransactionHeader {
 
     /** expiration of the transaction */
     expiry: TransactionExpiry.Type;
+
+    /** energy amount */
+    energyAmount: Energy.Type;
```

Comment:
Consider renaming to `baseEnergyAmount` and document that this excludes transaction size and signature costs. Or possibly not "base" because that is also used to refer to the transaction size + signatures part of the cost.